src/main/kotlin/frc/robot/subsystems/superstructure/Intake.kt

```diff
@@ -1,5 +1,64 @@
 package frc.robot.subsystems.superstructure
 
 import org.ghrobotics.lib.commands.FalconSubsystem
+import org.ghrobotics.lib.mathematics.units.nativeunit.DefaultNativeUnitModel
+import org.ghrobotics.lib.motors.ctre.FalconSRX
+import org.ghrobotics.lib.wrappers.FalconDoubleSolenoid
+import org.ghrobotics.lib.wrappers.FalconSolenoid
+
+
+var cargo = false
+object Intake : FalconSubsystem() {
+    val hatchMotor = FalconSRX(51, DefaultNativeUnitModel)
+    val cargoMotor = FalconSRX(50, DefaultNativeUnitModel)
+    val solenoid = FalconDoubleSolenoid(0,1, 8)
+
+        //No u, from gru
+
+
+    fun cargoIntake() {
+        cargoMotor.talonSRX.configPeakOutputForward(0.8)
+        hatchMotor.talonSRX.configPeakOutputForward(0.8)
```

Comment:
Ok so all that this does it set the maximum output that the motor can be commanded to 80%. So even if you tell it to go 100% duty cycle it'll only send 80% to the motor. You want cargoMotor.setDutyCycle instead. And make sure that the lateInit function calls `motor.talonSRX.configFactoryDefault()` for cargo and hatch motors

Reply:
done, i made them only yeet because we only yeet them

Reply:
que

Reply:
illegal move, do it right